src/server.js

```diff
@@ -11,7 +11,9 @@ function runServer(databaseUrl, port = PORT) {
   return new Promise((resolve, reject) => {
     // mongoose.set('debug', true);
     mongoose.set("useCreateIndex", true);
-    mongoose.connect(databaseUrl, { useNewUrlParser: true }, err => {
+    //DeprecationWarning: current Server Discovery and Monitoring engine is deprecated, and will be removed in a future version. 
+    //To use the new Server Discover and Monitoring engine, pass option { useUnifiedTopology: true } to the MongoClient constructor.
```

Comment:
I don't think we need the reason in here, I think you could probably delete it.  This could be a sperate PR, saying something like "passing option "useUnifiedTopology" to mongoose.connect to clear deprecation warning"

Reply:
I can do that